WebDriverAgentLib/Routing/FBElementCache.m

```diff
@@ -26,14 +27,14 @@ - (instancetype)init
   if (!self) {
     return nil;
   }
-  _elementCache = [[NSMutableDictionary alloc] init];
+  _elementCache = [[LRUCache alloc] initWithCapacity:1024];
```

Comment:
Maximum capacity should be a constant. Also, I'd like to see a comment there about how much memory a single record requires, so we can approximately calculate max cache size

Reply:
OK - done. Each record requires +/- 100KB of memory in my testing, so 1024 * 100 KB = 100 MB. Which is a lot for an element cache, but seems reasonable given the amount of memory iPhones and iPads have these days.